docs-source/content/quickstart/prepare.md

```diff
@@ -12,24 +12,22 @@ weight: 5
     $ kubectl create namespace sample-domain1-ns
     ```
 
-1.	Use `helm` to configure the operator to manage domains in this namespace:
-
+2.	Use `helm` to configure the operator to manage domains in this namespace:
```

Comment:
The original step numbering did not need to be changed. Even when you only use the number 1, the correct numbering sequence appears.

Reply:
I was wondering why it is like that, should we change it back ?

Reply:
I did.